src/components/cards/CardsGrid.tsx

```diff
@@ -25,6 +26,7 @@ const CardsGrid = (props: Props) => {
       >
         {props.items.map((prop) => (
           <AwardCard
+            key={uuidv4()}
```

Comment:
If this is coming from a backend, wouldn't an ID be there? if not, you can use an index. I don't like the use of uuids for stuff like this